docs/application/dotnet/guides/wcircularui/quickstart.md

```diff
@@ -0,0 +1,145 @@
+# Quickstart
+
+This guide explains how to create an application using **Toast** and **CircleSlider** UI contols. 
+
+- When you press the button, **Toast** pops up and it automatically disappears when the timer timeout (in seconds).
+- When you rotate the bezel of Tizen wearable device, **CircleSlider** bar moves either in forward or in backward direction.
+
+## Prerequisite
+
+- Visual Studio
+- [Visual Studio tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen)
+
+## Set Up Development Environment and Create Project
+
+Following are the steps to create a Tizen Wearable App Template:
+
+- [Create Tizen Xamarin.Forms](#create-tizen-xamarin-forms-application)
+- [Create Project](#create-project)
+- [Insert CircularUI Control Code](#insert-circularui-control-code)
+- [Build and Launch Your Application](#build-and-launch-your-application)
+
+### Create Tizen Xamarin Forms Application
+
+Create a Tizen Xamarin Forms application. For more information, see [Creating Your First Tizen .NET Application](https://developer.tizen.org/development/training/.net-application/creating-your-first-tizen-.net-application).
```

Comment:
Please change link to `[Creating Your First Tizen Wearable .NET Application](../../getting-started/wearable/first-app.md)`. It is leading to old page not used now.

Also, Please check : I choosed "Creating Your First Tizen **Wearable** .NET Application" among mobile, wearable, and TV.
Is it correct to choose wearable?

Reply:
I remove this chapter.